tests/fstab_test.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+from storage_tests.storagetestcase import StorageTestCase
+
+import blivet
+
+FSTAB_WRITE_FILE = "/var/tmp/test-blivet-fstab"
```

Comment:
`/var` is persistent, probably not a best place to add temporary files. Using the `tempfile` module to create a temporary directory would be better.

Reply:
Changed path to /tmp/...
I am not using `tempfile` on purpose. Created file has to be closed before blivet modifies it. Closing the `tempfile` will immediately remove it.

Reply:
The tempfile library also allows creating temporary directories, see https://docs.python.org/3.8/library/tempfile.html#tempfile.TemporaryDirectory I'd prefer using that rather than a custom cleanup.

Reply:
Changed to TemporaryDirectory.